server/auto_merge.go

```diff
@@ -105,19 +107,41 @@ func (s *Server) AutoMergePR() error {
 			MergeMethod: "squash",
 		}
 
+		if translationPr {
+			opt.MergeMethod = "merge"
+		}
+
 		merged, _, err := s.GithubClient.PullRequests.Merge(ctx, pr.RepoOwner, pr.RepoName, pr.Number, "Automatic Merge", opt)
 		if err != nil {
 			errMsg := fmt.Sprintf("Error while trying to automerge the PR\nErr %s", err.Error())
 			if err = s.sendGitHubComment(ctx, pr.RepoOwner, pr.RepoName, pr.Number, errMsg); err != nil {
 				mlog.Warn("Error while commenting", mlog.Err(err))
 			}
+			if err = s.removeTranslationLabel(ctx, pr); err != nil {
```

Comment:
Is the idea to remove the label if merging fails to avoid retrying? Should we gate this to only run if `translationPr`?

Reply:
Yes by removing that label, mattermod will not retry again. It will create a comment on PR and send notification to mattermost with releaseteam mention. So it can be easily fixed by release team.

+1 for `translationPr` gate. I will update codebase.

Reply:
Fixes are applied into codebase.